src/main/java/teammates/storage/sqlentity/FeedbackSession.java

```diff
@@ -152,12 +155,15 @@ public List<String> getInvalidityInfo() {
         addNonEmptyError(FieldValidator.getInvalidityInfoForTimeForVisibilityStartAndResultsPublish(
                 actualSessionVisibleFromTime, resultsVisibleFromTime), errors);
 
-        // TODO: add once extended dealines added to entity
-        // addNonEmptyError(FieldValidator.getInvalidityInfoForTimeForSessionEndAndExtendedDeadlines(
-        // endTime, studentDeadlines), errors);
+        // Fetches the DeadlineExtensions associated with this (feedback session) id
+        List<DeadlineExtension> deadlineExtensions = DeadlineExtensionsDb.inst().getDeadlineExtensionsBySessionId(id);
+        Map<String, Instant> deadlines = new HashMap<>();
+        for (DeadlineExtension de : deadlineExtensions) {
+            deadlines.put(de.getUser().getEmail(), de.getEndTime());
+        }
 
-        // addNonEmptyError(FieldValidator.getInvalidityInfoForTimeForSessionEndAndExtendedDeadlines(
-        // endTime, instructorDeadlines), errors);
+        addNonEmptyError(FieldValidator.getInvalidityInfoForTimeForSessionEndAndExtendedDeadlines(
+                endTime, deadlines), errors);
```

Comment:
Also add this to invalidity of deadline extensions.

There's some architectural issues with validation that we will solve another time, this will suffice for now.